packages/analytics/dashboard-renderer/src/components/BaseAnalyticsChartRenderer.vue

```diff
@@ -50,6 +54,26 @@ const options = computed((): AnalyticsChartOptions => ({
   chartDatasetColors: props.chartOptions.chartDatasetColors,
 }))
 
+const exploreLink = computed(() => {
+  if (queryBridge && queryBridge.exploreBaseUrl) {
+    const exploreQuery: ExploreQuery | AiExploreQuery = {
+      filters: props.query.filters?.map((filter) => {
+        return {
+          ...filter,
+          type: filter.type,
+        } as ExploreFilter
+      }) ?? [],
+      metrics: props.query.metrics as ExploreAggregations[] | AiExploreAggregations[] ?? [],
+      dimensions: props.query.dimensions as QueryableExploreDimensions[] | QueryableAiExploreDimensions[] ?? [],
+      time_range: props.query.time_range as TimeRangeV4 || props.context.timeSpec,
+
+    } as ExploreQuery | AiExploreQuery
+    return `${queryBridge.exploreBaseUrl()}?q=${JSON.stringify(exploreQuery)}&d=${props.query.datasource}&c=${props.chartOptions.type}`
```

Comment:
Sorry I missed this earlier: `props.query.datasource` might not be populated at this stage; it's not required in the schema yet and the fallback is in `QueryDataProvider`.  You can probably do `props.query.datasource || 'advanced'` in this case.

Reply:
done